frontend/src/components/elements/Text/Text.tsx

```diff
@@ -28,10 +32,21 @@ export interface TextProps {
  */
 export default function Text({ width, element }: TextProps): ReactElement {
   const styleProp = { width }
-
-  return (
+  return element.body.length > 40 ? (
+    <StyledLabelHelpWrapper>
+      <StyledText data-testid="stText" style={styleProp}>
+        {element.body}
+      </StyledText>
+      {element.help && (
+        <InlineTooltipIcon content={element.help}></InlineTooltipIcon>
+      )}
+    </StyledLabelHelpWrapper>
+  ) : (
     <StyledText data-testid="stText" style={styleProp}>
       {element.body}
+      {element.help && (
+        <InlineTooltipIcon content={element.help}></InlineTooltipIcon>
+      )}
```

Comment:
FWIW it would be better to completely remove conditional logic by avoiding changes to `StyledText` styling and remove the style prop passed to that element (the width). 
As a means to achieve the intended behavior more simply, I think you could use the `StyledLabelHelpWrapper` as your flexed parent and pass width to it instead :
```javascript 
export default function Text({ width, element }: TextProps): ReactElement {
  const styleProp = { width }
  return (
    <StyledLabelHelpWrapper style={styleProp}>
      <StyledText data-testid="stText">{element.body}</StyledText>
      {element.help && <InlineTooltipIcon content={element.help} />}
    </StyledLabelHelpWrapper>
  )
}
```

Reply:
Your solution is the best, thank you! 🙇 PR is updated with it 👍